extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/AuthenticationRedirectException.java

```diff
@@ -20,7 +20,7 @@ public AuthenticationRedirectException(int code, String redirectUri) {
     }
```

Comment:
This is the correct version of this file. I removed the constructor I had added in a previous version.